internal/git/commands_test.go

```diff
@@ -1158,4 +1158,50 @@ func TestBackendCommands(t *testing.T) {
 			must.EqOp(t, want, have)
 		})
 	})
+
+	t.Run("MergeFastForward", func(t *testing.T) {
```

Comment:
Done. I also added a comment on line 20 to highlight the sorting. I completely missed that this is supposed to be sorted since all sorting keys (the first `t.Run` argument) are so far apart from each other.

PTAL.